run_bonsai.sh

```diff
@@ -0,0 +1,9 @@
+export BONSAI_API_KEY="1234"
+export BONSAI_API_URL="https://api.bonsai.xyz/"
+# https://dev.risczero.com/api/blockchain-integration/contracts/verifier
+# RiscZeroGroth16Verifier.sol Sepolia	0x83C2e9CD64B2A16D3908E94C7654f3864212E2F8
+export GROTH16_VERIFIER_ADDRESS="0x83C2e9CD64B2A16D3908E94C7654f3864212E2F8"
+export GROTH16_VERIFIER_RPC_URL="https://any sepolia endpoint"
```

Comment:
I'm assuming this is not a valid RPC URL haha. If you make it part of the prove_block script it would be great to set this to the correct value depending on the selected chain.

Reply:
I think we don't have common sepolia node, is it? so I assumed each runner has own sepolia node service. anyway I can setup an infura entry here.

Reply:
Oh I see, I just assumed there would be one like we have for katja on holesky for example, or some free one that is available somewhere. If not, no need to set anything up I'd say.

Reply:
infura is kind of free, daily quota 100000 which should be enough for this purpose. updating.

Reply:
I deployed one in our devnet, and updated the addr & rpc url, now we can use either devnet rpc or sepolia.